### ISSUE TITLE
'Open in External Browser' is not discoverable in the appropriate cognitive contexts

### ISSUE THREAD

**Issue by ssokolow** [desura/Desurium#137]
In the internal browser, I often find myself wanting features only present in full-fledged browsers (like "Save As..." in the context menu for images, getting the link for a page without a Share button, or opening a link without leaving the current page). Currently, my workaround for all of these problems is to use "View Source" and then manually extract the URL I care about. (Obviously _not_ ideal)

I think the simplest way to solve all three of my example problems, plus other potential ones, is to make two small changes:
- Middle-clicking a link should open it in the user's preferred external browser via xdg-open.
- The context menu should have an option to send the current page to the user's browser.

---

**Original title:** No simple way to open a page in an external browser.

Since posting this bug, I have been informed that the functionality does exist... but only as a button containing the "link to external site" icon (as popularized by Wikipedia/MediaWiki) next to the search box. Therefore, to someone who either doesn't exhaustively inspect every UI element or misses it or forgets about it and checks the context menu and middle-click functionality out of habit, it's still not discoverable.

**Comment by Jookia** [comment]
I could've sworn this was discussed before. Click the 'pop-out' button next to the search bar.

**Comment by ssokolow** [comment]
Huh... and I thought I'd checked the tooltips on all visible UI elements already.

Perhaps we could change the focus of this bug to "'open in external browser' is not discoverable in the appropriate cognitive contexts"?

**Comment by Jookia** [comment]
I'll agree to that.

**Comment by mathrick** [comment]
I'm pretty confused, but isn't that already possible? In the official non-beta Windows client, right-clicking anywhere gives you "View Page in Browser" as the last menu position. Additionally, middle-clicking links in the forum (but _only_ there, as far as I'm able to tell) opens them in the external browser. So it might be that the real issue here is getting feature parity and consistency more than getting any single one implemented.

**Comment by ssokolow** [comment]
In the GPLed or the official, non-beta Linux client, the context menu is as follows:

```
Back
Forward
Reload
----
Zoom In
Zoom Out
Zoom Normal
----
View Source
```

I rarely go to the forums, but middle-clicking behaves the same as left-clicking in the Games and Community tabs.

**Comment by mathrick** [comment]
On Apr 4, 2012 7:23 AM, "Stephan Sokolow" <
reply@reply.github.com>
wrote:

> Even in the official, non-beta Linux client, the context menu is as
> follows:
> 
>    Back
>    Forward
>    Reload
> 
> ---
> 
>    Zoom In
>    Zoom Out
>    Zoom Normal
> 
> ---
> 
>    View Source

Precisely. Thus I don't understand what this issue is about.

> I rarely go to the forums, but middle-clicking behaves the same as
> left-clicking in the Games and Community tabs.

Yes. So they'd need to be brought in line with the forum, which already
does what we want.

**Comment by ssokolow** [comment]
Makes sense... though I'm not sure how only parts of a website would be able to trigger what should be a privileged function.
